spec/models/pending_purchase_spec.rb

```diff
@@ -1,7 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe PendingPurchase, type: :model do
-  let(:pending_purchase) { FactoryGirl.create(:pending_purchase) }
+  let(:user) { FactoryGirl.create(:user) }
```

Comment:
Protip: If you add `config.include FactoryGirl::Syntax::Methods` into the `RSpec.configure` in `spec_helper.rb`, you can drop the `FactoryGirl.` everywhere.